docs/nitpick-exceptions.ini

```diff
@@ -18,6 +18,10 @@ class_references=
     Setter
     Getter
     ; - AwsXRayFormat.extract
+    httpcore.SyncHTTPTransport
+    httpcore.AsyncHTTPTransport
+    httpcore.SyncByteStream
+    httpcore.AsyncByteStream
```

Comment:
instead of this, httpcore should be added to docs dependencies.

Reply:
It's a transitive dependency for `httpx`, and that is in the doc requirements. I dug in further and think that it's because of `httpcore` [changing `__module__`](https://github.com/encode/httpcore/blob/ed76a30277529c5687cbdb6e203d6dbd0d56833f/httpcore/__init__.py#L60), which prevents Sphinx from finding the actual locations. Is there an another alternative that you suggest?

Reply:
Looks like they have a [new site using Sphinx](https://www.encode.io/httpcore/), so I can use that with intersphinx. However, I will have some changes that use `httpx` classes, and so I can't do the same thing since that doc site isn't using Sphinx (yet?).

Reply:
OK, makes sense. 👍